cipherscan

```diff
@@ -1313,6 +1299,12 @@ do
             DELAY=$2
             shift 2
             ;;
+        --cafile)
+            CACERTS="$2"
+            shift 2
+            # We need to bypass autodetection if this is provided.
+            CACERTS_ARG_SET=1
```

Comment:
I'm not sure what's the purpose of adding a new variable

 * if the cafile and capath is set at the same time, we can just abort
 * if capath is not a directory, error out
 * if capath is empty and cacerts is empty - do autodetection of cacerts
 * if cacerts is still empty or not points at a readable file, error out that there's no trust store available